app/models/mission.rb

```diff
@@ -2,8 +2,9 @@
 
 class Mission < ApplicationRecord
   validates :title, presence: true, length: { maximum: 50 }
-  validates :content, presence: true
+  validates :content, :status, presence: true
   validate :validate_end_time_after_start_time
+  enum status: { pending: 0, processing: 1, completed: 2 }
```

Comment:
Nit: 為了閱讀方便，是我的話會將「定義屬性的 `.enum` 」與「驗證用的 validation methods 」之間空行。
```suggestion
  validate :validate_end_time_after_start_time

  enum status: { pending: 0, processing: 1, completed: 2 }
```